google/cloud/security/common/gcp_type/gce_network.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""A Enforced Network Resource."""
+
+
+# pylint: disable=too-few-public-methods
+class GceNetwork(object):
+    """Enfoced Networks Resource."""
+
+    def __init__(self, project, network, is_external_network, whitelist):
+        """Initialize
+
+        Args:
+            project: string of gcp project
+            network: network in the projct
+        """
+        self.project = project
```

Comment:
The attributes here seem to match the rules attribute?  Instead, these attributes should match the the resource attribute.
https://cloud.google.com/compute/docs/reference/beta/instances#resource

![image](https://user-images.githubusercontent.com/6977544/27407432-19a2b224-568d-11e7-9861-8084505193e0.png)